src/models.py

```diff
@@ -163,7 +157,7 @@ class Condition(Base):
     created_at = Column(
         DateTime(timezone=True), default=datetime.datetime.utcnow(), nullable=False
     )
-
+    is_active = Column(Boolean, nullable=False)
```

Comment:
So you can activate or not a condition ?

Reply:
The idea is that conditions may expire for whatever reason, and should not be taken into account later on.